_posts/2020-03-15-learning-to-share-accomplishments-2.md

```diff
@@ -0,0 +1,214 @@
+  experience. (It's similar to Haskell, which I had done a project on in the past, so I picked it up quickly.)
+* A teammate couldn't get one of our services working in Scala so they rewrote it all in Ruby over a weekend. We ended 
+  up keeping that, but it was built to run locally. After that person quit shortly after, I Dockerized it, wrote the CI 
+  scripts to deploy it to AWS, and got it to work with AWS's Aurora databases (as opposed to a local Postgres install). 
+  I later ended up resolving at least 85% of the security flaws that came up in a security audit on it. I did this 
+  without prior Ruby or Ruby on Rails knowledge, which I picked up quickly).
+* When I was hired, they wanted me to start work immediately and relocate to Pittsburgh as I could. I started work 
+  within about two weeks of the offer, and spent the first two months working remote out of Kansas City (where the time 
+  zone was an hour earlier than my team). I also was packing to move at the same time I did this work.
+* From day one, I was given ownership of two of the 8-10 microservices our project had.
+* I wrote and modified existing Terraform scripts to add additional deploy and containerization features for some of 
+  our microservices. (I had never used Terraform before either.)
+* After our original manager was fired and a new one was brought in, I regularly checked in on and mentored the junior 
+  member of our team when the new manager didn't take that on.
+* We didn't have 1-on-1 meetings after our original manager wasn't there anymore. I tried to have unofficial 1:1s with 
+  the two original remaining team members just to make sure they were doing ok. 
+
+### Favorite Work Project: Team Metric Tracker
+
+I really liked this project not because it was particularly complex or really cool, but because of the creative 
+solutions that came out of it.
+
+Our five-person team was all hired at the same time. We were on a three month internal project which helped us learn 
+the security protocols for software at the bank. The project was sold to us as "make this Excel spreadsheet into a web 
+app". Once we dived in deep, we realized this was a quirky problem with some hidden issues. The spreadsheet had a 
+dozen tabs, data from several years, a constantly evolving report, and a dozen users that all added data to it. Our 
+basic web app turned into a quirky data project that required data auditing, a report-designing system, hierarchies of 
+management, and redefining terminology the department used. (For more fun, we were told another team tried to tackle 
+this in the past and couldn't do it.)
+
+**What did I Do?**
+
+* While many of us worked on the original design and wireframes, I came up with the idea of redefining their terminology 
+  (what we called "metrics" and different kinds of metrics) so it made sense to us (the client used a lot of words 
```

Comment:
This is the sort of thing I would do and absolutely not count as a contribution later! 👏

Reply:
Count it, Veronica!